packages/pulp-deb/pulp-deb.spec

```diff
@@ -125,7 +124,9 @@ client capabilites with Debian specific features.
 %doc COPYRIGHT LICENSE AUTHORS
 
 %changelog
-* Wed Oct 11 2017 Bernhard Suttner <suttner@atix.de> 1.5.2-2                                  
+* Fri Jun 07 2019 Quirin Pamp <pamp@atix.de> 1.10.0-?
```

Comment:
Not sure what to put for the release version here, since it is `0.1.alpha%{?dist}` in the spec file above.
Any help is appreciated...

Reply:
I am guessing I should also update the spec file version/release version somehow (just not sure what I should change it to).